src/app/models/propTypes/index.js

```diff
@@ -41,12 +41,15 @@ export const videoPropTypes = {
 };
 
 export const videoComponentPropTypes = {
-  duration: string,
-  imageLocator: string,
-  kind: string,
-  rawImageSrc: string,
-  versionID: string,
-  videoLocator: string,
+  pid: string.isRequired,
+  kind: string.isRequired,
+  title: string.isRequired,
+  items: shape({
```

Comment:
It's great that you've updated these proptypes as well. 

`items` should be an array of such objects. This is since a video player can have multiple items passed to it. Can you please update this and the associated logic in the Video Container?